cockatrice/src/window_main.cpp

```diff
@@ -334,10 +337,10 @@ void MainWindow::loginError(Response::ResponseCode r, QString reasonStr, quint32
     switch (r) {
         case Response::RespClientUpdateRequired: {
             QString formattedMissingFeatures;
-            formattedMissingFeatures = "Missing Features: ";
+            formattedMissingFeatures = "tr("Missing Features:") + QString:(" ")";
```

Comment:
Thanks. That should be ok now. 👍 
But well, Travis/Appveyor throws a lot of other stuff too...